git-log-cart-from-dec7-to-dev20.txt

```diff
@@ -0,0 +1,441 @@
+BRIAN DONE - commit 3cddd19c910b224b834d09d29920698408be704b
```

Comment:
left here if you want to double check things

- left keywords with BRIAN to see the related commits touching checkout
- left keywords with ======= START ======= and ======= END ======== that gets auto merged so not relevant